app/src/main/java/com/SecUpwN/AIMSICD/smsdetection/SmsDetector.java

```diff
@@ -504,4 +313,69 @@ public void onServiceDisconnected(ComponentName arg0) {
             mBound = false;
         }
     };
+
+    private String findSmsData(String[] prebuffer,String[] postbuffer){
```

Comment:
>All you seem to do is add more whitespace and delete valuable comments and code that is needed for future work

did you even bother checking my contributions to this project? Comments and commented code should never be pushed to master branch. This is a public security project where code quality must be kept on highest level.

>You seem to find plenty of time to point out stuff but no time to fix important issues which someone of your coding ability could fix in 5 mins

I did, check my commits. I did a lot of good changes with good (not best) but good quality code.